src/main/scala/mesosphere/marathon/core/launcher/impl/OfferProcessorImpl.scala

```diff
@@ -57,7 +58,10 @@ private[launcher] class OfferProcessorImpl(
       .recover {
         case NonFatal(e) =>
           matchErrorsMeter.mark()
-          log.error(s"error while matching '${offer.getId.getValue}'", e)
+          val message =
+            if (e.isInstanceOf[AskTimeoutException]) "in time. Extend with --max_offer_matching_timeout."
+            else s"${e.getClass.getSimpleName}: ${e.getMessage}"
+          log.warn(s"Could not process offer '${offer.getId.getValue}' $message")
```

Comment:
And can we maybe generalize this code to apply it elsewhere where we ask with a timeout?